### ISSUE TITLE
in game issue

### ISSUE THREAD

**Issue by Sancty333** [thomas-barthelemy/QQ-LoL-Localizer#12]
sorry to bother but since the client changed I have issues in game. Actually the client is almost ok all is in english but when I go in the shop I can't see anymore the week promotions. About the in gam it is back in chinese even if the client is in english. I hope you can help me with this. thank you very much for your time

**Comment by jediofthecode** [comment]
I am also having the same issue, it seems the new game client is not using the same files for language.

**Comment by thomas-barthelemy** [comment]
Hey, thanks for the feedback. Can you both confirm the version of the localizer you are using?
But if confirmed on all latest versions it would be quite sad to see that they are changing their file architecture every 2 versions...

**Comment by Sancty333** [comment]
I tried something you solved in august and now all is in english but I don't have the voice or music anymore (finish the sexy voice saying PENTA kill haha) and I am on the version 2.10.0.0 as you said in the august report

**Comment by Sancty333** [comment]
when I said no voices it includes champions voices too

**Comment by thomas-barthelemy** [comment]
latest release is 2.14:
https://github.com/thomas-barthelemy/QQ-LoL-Localizer/releases/tag/v2.14

give it a try =)

**Comment by Camineet** [comment]
Hi Thomas,

I just downloaded the new version you posted a few minutes ago.  Thanks for
your ongoing work to help us non-Chinese speakers.

Launching the new version, it says my game is fixed without even having to
fix it by clicking the fix button.  It just says it's already fixed.

Unfortunately, my game is still in Chinese.

Even the manual methods of fixing the in-game text aren't working for me.

I have followed this guide exactly:

https://www.reddit.com/r/leagueoflegends/comments/3ct7tm/guidehow_to_change_the_chinese_lol_language/

I even uninstalled and reinstalled the game to see if I could try again to
change the language manually, but my game is still in Chinese although the
launcher is in English.

Any ideas you have about what else I can try would be much appreciated😀





Marc Joseph, MBA, LMSW

(248) 579-0856

On Sun, Oct 30, 2016 at 6:40 PM, Thomas Barthelemy <notifications@github.com

**Comment by thomas-barthelemy** [comment]
The link you mentioned on reddit is largely outdated and definitely won't work anymore. The best I could suggest right now is to uninstall (be sure all files are deleted correctly) and re-install and use the latest version to fix it.

Detecting whether the current files are fixed or not is really unstable and generally only works if fixed by the previous/current version of the localizer. Any manual change might not be understood properly.

Bottom-line: I'm 99% sure that there are no instruction to fix the client manually that are up-to-date anywhere, so following things you find will most likely break things more than anything else =(
It's quite sad to see how frequently they change the file structure, makes it a pain for us definitely.

**Comment by Camineet** [comment]
ok thanks so much for the additional instructions.  I'll go ahead and
uninstall, reinstall, and then apply today's new version of localizer😀

Marc Joseph, MBA, LMSW

(248) 579-0856

On Sun, Oct 30, 2016 at 7:00 PM, Thomas Barthelemy <notifications@github.com

**Comment by Sancty333** [comment]
I am givin it a try right now. thank you very much for all your are doing for us stuck unable to play an eu or na version of the game .

**Comment by Camineet** [comment]
yeah i'm grateful too.  just arrived in xiamen and have played for almost 4
years.  the game feels a bit lifeless without the english voices even
though i have some pretty good mandarin skills.

i can deal with the chinese voice overs which are pretty awful in my
opinion, but not having the item and ability descriptions in english is
just miserable



Marc Joseph, MBA, LMSW

(248) 579-0856

**Comment by Sancty333** [comment]
well just tried this version and I didn't notice anything wrong in game. voices are back items in english . Just the shop were the offers are still invisible for no reason but actually it really doesn't bother me at all. thank you very much for improving my game experience.
Jordan from shanghai

**Comment by thomas-barthelemy** [comment]
I'm not really aware of what are the week promotions as it wasn't a thing the last time I played (years ago). If it's a Chinese only thing there is a good chance it may never get fixed. If not it's something that may be fixable =) Do let me know a little more about that.

**Comment by Camineet** [comment]
Not having any luck and would be grateful for some more help.

I have just now completed the following steps.

Uninstalled the game

Deleted the game folder from program files

Reinstalled game

Started a game to make sure anything missing from the installation package would be downloaded

Shutdown the game and closed TP application that runs in the background

Opened localizer which reported my game is not fixed

clicked 'Fix' and localizer reported my game is fixed

Started a new game and the launcher and in-game text is still Chinese as seen in attached screenshot even though all files are shown as green in localizer

Would be grateful for more help  :*


![capture](https://cloud.githubusercontent.com/assets/10620070/19836898/9db16522-9ee8-11e6-9573-7028805198a4.PNG)
![screen01](https://cloud.githubusercontent.com/assets/10620070/19836899/a357ac0c-9ee8-11e6-863d-3adb7e9d963e.png)

**Comment by Camineet** [comment]
tried running as administrator, which was not available in last localizer version at least for me, and finally the whole thing is english including in-game xD

now if i could also get the english voice acting in-game....

maybe that's too much to hope for

well, there's always heroes of the storm.  blizzard is such a better company.  not only do they offer english play in asia without all this extra hacking, but the game also matches you with other english speakers in the region

**Comment by Camineet** [comment]
tried running as administrator, which was not available in last localizer version at least for me, and finally the whole thing is english including in-game xD

now if i could also get the english voice acting in-game....

maybe that's too much to hope for

well, there's always heroes of the storm.  blizzard is such a better company.  not only do they offer english play in asia without all this extra hacking, but the game also matches you with other english speakers in the region

**Comment by Camineet** [comment]
anyway, thanks a lot for making this application Thomas :*

**Comment by thomas-barthelemy** [comment]
> now if i could also get the english voice acting in-game....
Sadly that's something that was stripped out of the game by the Tencent team as it was taking a lot of space for no reason. So there is not much magic possible here =(

> Started a new game and the launcher and in-game text is still Chinese as seen in attached screenshot even though all files are shown as green in localizer

Try to do a 
 1. `Restore all`
 2. Restart the localizer but as Administrator this time
 3. `Fix All`
 4. `Play Now` from the localizer

Although I find it very weird that it's able to `fix` without actually doing anything. From what is done, things should be either in English or Messed up but not just back in Chinese.

**Comment by Camineet** [comment]
yes, in game text is in english now.  thank you Thomas :D
